indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -717,6 +720,8 @@ public void onFailure(Throwable t)
               // here for kafka is to +1 while for kinesis we simply save the current sequence number
               lastReadOffsets.put(record.getPartitionId(), record.getSequenceNumber());
               currOffsets.put(record.getPartitionId(), getNextStartOffset(record.getSequenceNumber()));
+
+              partitionsThroughput.computeIfAbsent(record.getPartitionId(), igonre -> new LongAdder()).increment();
```

Comment:
One streaming record can have multiple rows (e.g. is delimited by a newline character), need to be sure which level is reporting on. IMO this need to report on rows level.